src/chrome/content/rules/escortradarforum.com.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="escortradarforum.com">
```

Comment:
please change this to `<ruleset name="escortradarforum.com" platform="mixedcontent>`